cadctap/cadctap/core.py

```diff
@@ -105,10 +103,13 @@
 ALLOWED_TB_DEF_TYPES = {'VOSITable': 'text/xml',
                         'VOTable': 'application/x-votable+xml'}
 AUTH_OPTION_EXPLANATION = \
-    'If no authentication option is specified, cadc-tap will look in the\n'\
-    '~/.netrc file for the cadc-ccda.hia-iha.nrc-cnrc.gc.ca or canfar.net\n'\
-    'domain, and if found, will use the -n option. If not, cadc-tap will\n'\
-    'look for ~/.ssl/cadcproxy.pem file, and if found, will use the --cert\n'\
+    '\nTo obtain the host associated with a service, execute a subcommand\n'\
```

Comment:
This blob repeats a few times. Please assign it to a variable and display that everywhere is necessary. That would be great for maintenance.

Reply:
The phrase 'host associated with a service' occurs twice. Do we really want to define a variable for that?